airmozilla/popcorn/templates/popcorn/editor.html

```diff
@@ -23,6 +23,7 @@
 {% endblock %}
 
 {% block full_width_content %}
+<slug data-url="{{ slug }}"></slug>
```

Comment:
I'm still not convinced we need this extra tag. We should be able to put this into the `<div id="editor">` tag.